pkgs/applications/misc/khard/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, pkgs, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  version = "0.3.3";
+  name = "khard-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/scheibler/khard/archive/v${version}.tar.gz";
+    sha256 = "06z5q5xlg4y79mb10klx97k0g71phga7c9m10vsm5fqicr38m488";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    configobj
+    vobject
+    argparse
+    pkgs.vdirsyncer
```

Comment:
Ah, forgot about this one. I must add the `vdirsyncer` package to the `buildDepends` list instead of the `propagatedBuildInputs` list, right?